plugins/woocommerce/client/admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/test/index.js

```diff
@@ -550,7 +574,7 @@ describe.skip( 'The message in the banner', () => {
 		);
 
 	const notActivatedMessage =
-		'By clicking "Create shipping label", WooCommerce Shipping(opens in a new tab) will be installed and you agree to its Terms of Service(opens in a new tab).';
+		'By clicking "Create shipping label", WooCommerce Shipping↗ will be installed and you agree to its Terms of Service↗.';
```

Comment:
The external link contains `↗` in rendered HTML and `aria-label` is `(opens in a new tab)`. 

https://wordpress.github.io/gutenberg/?path=/docs/components-externallink--docs